src/MapBuilder.js

```diff
@@ -42,7 +42,7 @@ export const MapBuilder = (props) => {
         initialValues={{
           lonField: fields[0].name,
           latField: fields[0].name,
-          geomField: null,
+          geomField: fields[0].name,
```

Comment:
@julietezekwe let's leave the initial value of `geomField` as `null` because if it's set it removes `lonField` and `latField` from the view spec. See https://github.com/datopian/map-builder/blob/master/src/MapBuilder.js#L30-L35

Reply:
Yea right, will do